src/integrations/Adroll/browser.js

```diff
@@ -17,6 +17,7 @@ class Adroll {
     this.analytics = analytics;
     this.advId = config.advId;
     this.pixId = config.pixId;
+    this.requiredConfigsList = requiredConfigsList;
```

Comment:
Why does it have to be defined in the class? We don't seem to use this.

Reply:
Removing